lib/server/_static.js

```diff
@@ -20,7 +20,7 @@ exports.create = async function() {
   var server = new Hapi.Server({
     host: config.server.host,
     port: config.server.port + 1,
-    debug: { request: ['error'] }
+    debug: false
```

Comment:
hm not sure about this change, I wonder what `{ request: ['error'] }` used to do

Reply:
it was originally false only